### PR TITLE
Adding new columns to show EB and CB org:hasSubOrganization relationship

### PR DESCRIPTION
**Pull request by Poltergeistz**
# Description
DL-5013

This PR adds `?worshipAdministrativeUnitRelationship` and `?worshipAdministrativeUnitRelationshipLabel` in report `link-between-worship-service-and-admin-units-report`. These variables will link worship administrative units with the rule EB has CB and show them in the report with the OPTIONAL statement. 

Report description is also changed in https://github.com/lblod/app-worship-decisions-database/pull/72/commits/3dba970c8fbc61bc3c87f937ce0dcf9d297a1b52

# Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other

# Related services

- report-generation

# How to test 

1. Add in `docker-compose.override.yml` 
```yml
  report-generation:
    links:
      - virtuoso:database  # we need to because performance
    ports:
      - 9999:80
```
3. `drc up -d`
4. Links between worship services and non-religious administrative units → 
```bash 
curl -X POST -H "Content-Type: application/vnd.api+json"  -d '{  "data": { "attributes": { "reportName": "linksBetweenWorshipServicesAndAdminUnitsReport" }  }  }'  http://localhost:9999/reports
```
5. Generated report = `{"data":{"type":"report-generation-tasks","attributes":{"status":"success"}}}` and should be visible in `/data/files/`

# What to check

- N/A

# Links to other PR's

- N/A

# Notes

- Update changelog

### CODE REVIEW

File: config/reports/links-between-worship-services-and-admin-units-report.js [hunk 1]
```diff
@@ -19,48 +19,53 @@ export default {
...
-            besluit:classificatie ?classification ;
-            ere:betrokkenBestuur ?betrokke .
-      
-          ?classification skos:prefLabel ?adminUnitClassification .
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+    PREFIX org: <http://www.w3.org/ns/org#>
+    PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+    PREFIX ere: <http://data.lblod.info/vocabularies/erediensten/>
+    
+    SELECT DISTINCT ?worshipAdministrativeUnit ?worshipLabel ?worshipClassification ?worshipAdministrativeUnitRelationship ?worshipAdministrativeUnitRelationshipLabel ?adminUnit ?adminUnitLabel ?adminUnitClassification ?typeLocalInvolvment
```
Comment (Riadabd) [inline config/reports/links-between-worship-services-and-admin-units-report.js:27]: Small typo: `typeLocalInvolvment` should be `typeLocalInvolvement`

```suggestion
    SELECT DISTINCT ?worshipAdministrativeUnit ?worshipLabel ?worshipClassification ?worshipAdministrativeUnitRelationship ?worshipAdministrativeUnitRelationshipLabel ?adminUnit ?adminUnitLabel ?adminUnitClassification ?typeLocalInvolvement
```

Comment (Poltergeistz) [reply config/reports/links-between-worship-services-and-admin-units-report.js]: Indeed, fixed :slightly_smiling_face:

File: config/reports/links-between-worship-services-and-admin-units-report.js [hunk 2]
```diff
@@ -82,7 +87,7 @@ export default {
       'adminUnit',
```
Comment (Riadabd) [inline config/reports/links-between-worship-services-and-admin-units-report.js:87]: This snippet is missing these two still, right?

```suggestion
      'worshipAdministrativeUnitRelationship',
      'worshipAdministrativeUnitRelationshipLabel'
      'adminUnit',
```

File: config/reports/links-between-worship-services-and-admin-units-report.js [hunk 3]
```diff
@@ -71,7 +76,7 @@ export default {
         adminUnit: unit.adminUnit.value,
```
Comment (Riadabd) [inline config/reports/links-between-worship-services-and-admin-units-report.js:76]: This is still missing the relevant lines `worshipAdministrativeUnitRelationship` and `'worshipAdministrativeUnitRelationshipLabel`.

File: config/reports/links-between-worship-services-and-admin-units-report.js (hunk 2 above)
Comment (Poltergeistz) [reply config/reports/links-between-worship-services-and-admin-units-report.js:91]: Yes you're right, not sure how I missed that when I tested it

**Review by Riadabd — APPROVED** [review]
Good to go :+1:

**Pull request merged**